Capstone-Projects-2022-Spring/project-vicsr
# Vic 153 UI vocabulary list floating to the right of document

Fully functioning vocabulary list

To Test:

1. Log in to an account
2. Upload a document with highlighted words (or use one already existing)
3. Select that doc
4. See that the associated words populate the list on the right of the screen
5. Click on a word (see onHover styling)
6. See a modal open that displays the associated backed translation and definition information
7. Close modal
